Signal/src/ViewControllers/ContactsPicker.swift

```diff
@@ -231,13 +219,16 @@ public class ContactsPicker: OWSViewController, UITableViewDelegate, UITableView
     // MARK: - Table View Delegates
 
     open func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: contactCellReuseIdentifier, for: indexPath) as! ContactCell
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: contactCellReuseIdentifier, for: indexPath) as? ContactCell else {
+            owsFail("\(logTag) in \(#function) cell had unexpected type")
+            return UITableViewCell()
+        }
 
         let dataSource = filteredSections
         let cnContact = dataSource[indexPath.section][indexPath.row]
         let contact = Contact(systemContact: cnContact)
 
-        cell.configure(contact: contact, subtitleType: subtitleCellValue, contactsManager: self.contactsManager)
+        cell.configure(contact: contact, subtitleType: subtitleCellType, showsWhenSelected: self.allowsMultipleSelection, contactsManager: self.contactsManager)
```

Comment:
This is the only meaningful change in this file - the rest are just for clarity.

We now tell the ContactCell whether the tableView allowsMultipleSelection so that we only show the ✔️ accessory view when using multiselect.

In the case of single-select contact picker, it felt a little weird to see the checkmark appear just as the view disappears.